### ISSUE TITLE
Handle koji messages from the messagebus plugin

### ISSUE THREAD

**Issue by ralphbean** [fedora-infra/pdc-updater#21]
Here are some examples from the non-fedmsg messagebus plugin in koji.

We'll need to modify pdc-updater to get some handlers working with these.

```python
Message Properties:
{u'old': u'OPEN', u'parent': 12215763, u'attribute': u'state', u'method': u'createrepo', u'new': u'CLOSED', u'type': u'TaskStateChange', u'id': 12215768}
Message Content:
{u'attribute': u'state',
 u'info': {u'arch': u'noarch',
           u'awaited': True,
           u'channel_id': 3,
           u'completion_time': u'2016-12-08 17:47:16.338368',
           u'completion_ts': 1481219236.33837,
           u'create_time': u'2016-12-08 17:45:27.826755',
           u'create_ts': 1481219127.82676,
           u'host_id': 182,
           u'id': 12215768,
           u'label': u'x86_64',
           u'method': u'createrepo',
           u'owner': 3371,
           u'parent': 12215763,
           u'priority': 14,
           u'request': [1813792,
                        u'x86_64',
                        {u'create_event': 14522694,
                         u'create_ts': 1481045117.11949,
                         u'creation_time': u'2016-12-06 17:25:17.119495',
                         u'id': 1811522,
                         u'state': 1}],
           u'result': [u'tasks/5768/12215768',
                       [u'b1e5e4b848fc6fc0d798a3617d2477e235c0cf5846f4f3707cb07a5210505d60-filelists.xml.gz',
                        u'repomd.xml',
                        u'f44697cd5c3509999eb5ebfce98f87ad07dd6dd69c99c65df9201224ef127f46-primary.sqlite.bz2',
                        u'184507f17c33a9f3f5723c5e214b305389a86a57d4beb300ebbae7101bb922d9-primary.xml.gz',
                        u'4a40a350242f2c273cf14d0f5e17bb799a90c8062be90409849b4a31574b358e-other.xml.gz',
                        u'e9437f9d43d7e5f19055ba3e355e1d40336be9992f8d31cf79ac80fa58df2573-comps.xml',
                        u'aa37c1149ae05adc75b1eb1da07fb7a6ecea3cd2f77f370db1125aadc330030b-other.sqlite.bz2',
                        u'd7ab35c7d24782ea33e81edc1d88a5464b478285e15bc972b46f3831561b8153-filelists.sqlite.bz2',
                        u'435547fc477f42d9ed4d76177bda4f0f64f85fe8de4545305def903f5c7b6900-comps.xml.gz']],
           u'start_time': u'2016-12-08 17:45:50.597412',
           u'start_ts': 1481219150.59741,
           u'state': 2,
           u'waiting': None,
           u'weight': 1.5},
 u'new': u'CLOSED',
 u'old': u'OPEN'}
```

```python
Message Properties:
{u'old': None, u'parent': 12215763, u'attribute': u'completion_ts', u'method': u'createrepo', u'new': 1481219236.33837, u'type': u'TaskStateChange', u'id': 12215768}
Message Content:
{u'attribute': u'completion_ts',
 u'info': {u'arch': u'noarch',
           u'awaited': True,
           u'channel_id': 3,
           u'completion_time': u'2016-12-08 17:47:16.338368',
           u'completion_ts': 1481219236.33837,
           u'create_time': u'2016-12-08 17:45:27.826755',
           u'create_ts': 1481219127.82676,
           u'host_id': 182,
           u'id': 12215768,
           u'label': u'x86_64',
           u'method': u'createrepo',
           u'owner': 3371,
           u'parent': 12215763,
           u'priority': 14,
           u'request': [1813792,
                        u'x86_64',
                        {u'create_event': 14522694,
                         u'create_ts': 1481045117.11949,
                         u'creation_time': u'2016-12-06 17:25:17.119495',
                         u'id': 1811522,
                         u'state': 1}],
           u'result': [u'tasks/5768/12215768',
                       [u'b1e5e4b848fc6fc0d798a3617d2477e235c0cf5846f4f3707cb07a5210505d60-filelists.xml.gz',
                        u'repomd.xml',
                        u'f44697cd5c3509999eb5ebfce98f87ad07dd6dd69c99c65df9201224ef127f46-primary.sqlite.bz2',
                        u'184507f17c33a9f3f5723c5e214b305389a86a57d4beb300ebbae7101bb922d9-primary.xml.gz',
                        u'4a40a350242f2c273cf14d0f5e17bb799a90c8062be90409849b4a31574b358e-other.xml.gz',
                        u'e9437f9d43d7e5f19055ba3e355e1d40336be9992f8d31cf79ac80fa58df2573-comps.xml',
                        u'aa37c1149ae05adc75b1eb1da07fb7a6ecea3cd2f77f370db1125aadc330030b-other.sqlite.bz2',
                        u'd7ab35c7d24782ea33e81edc1d88a5464b478285e15bc972b46f3831561b8153-filelists.sqlite.bz2',
                        u'435547fc477f42d9ed4d76177bda4f0f64f85fe8de4545305def903f5c7b6900-comps.xml.gz']],
           u'start_time': u'2016-12-08 17:45:50.597412',
           u'start_ts': 1481219150.59741,
           u'state': 2,
           u'waiting': None,
           u'weight': 1.5},
 u'new': 1481219236.33837,
 u'old': None}
```

**Comment by ralphbean** [comment]
@mprahl was going to look at gathering some better examples.  We need a `'Tag'` message, in particular.

**Comment by ralphbean** [comment]
Closed by #22.
